pluto_build/05_export.sh

```diff
@@ -55,9 +55,3 @@ mkdir -p output/dof &&
   )
 
 wait
```

Comment:
should this wait still be here?

Reply:
I think the wait at the beginning of the upload file is probably accomplishing the same thing?

Reply:
I assumed that when the execution got to that `wait` it would check to make sure all background processes were done. So having it below the export step has it check that the export child processes finish. But maybe that's not actually how it works?

Reply:
Ok, that makes sense